ci-operator/step-registry/interop/mtr/OWNERS

```diff
@@ -0,0 +1,17 @@
+approvers:
```

Comment:
We should add Shveta to the approvers and reviewers list for the MTR specific files, including the config once we move it to the windup dir.

Reply:
Agreed, I have updated the org to the windup org rather than my own fork for the repository. I will add Shveta as an owner on the MTR specific files.